corehq/tests/optimizer.py

```diff
@@ -122,6 +119,8 @@ def _extract_tests(test_suite_or_test):
 @nottest
 @contextmanager
 def optimize_apps_for_test_labels(test_labels):
+    # TODO make this work on Django 1.8 (django.test.simple has been removed)
```

Comment:
i'm good removing. I also haven't used the optimizer since the brave new `REUSE_DB` world, and it's caused random issues with people running tests too.